packages/core/src/actions/prepareTransactionRequest.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  type PrepareTransactionRequestErrorType as viem_PrepareTransactionRequestErrorType,
+  type PrepareTransactionRequestParameters as viem_PrepareTransactionRequestParameters,
+  type PrepareTransactionRequestReturnType as viem_PrepareTransactionRequestReturnType,
+  prepareTransactionRequest as viem_prepareTransactionRequest,
+} from 'viem/actions'
+
+import { type Config } from '../createConfig.js'
+import { type ChainIdParameter } from '../types/properties.js'
+import { type Evaluate, type UnionLooseOmit } from '../types/utils.js'
+
+export type PrepareTransactionRequestParameters<
+  config extends Config = Config,
+> = Evaluate<
+  UnionLooseOmit<viem_PrepareTransactionRequestParameters, 'chain'> &
```

Comment:
We should make `to` required (similar to how it is set up in [`sendTransaction`](https://github.com/wevm/wagmi/blob/main/packages/core/src/actions/sendTransaction.ts#L41)). We also need to wire up some generic slots for [`viem_PrepareTransactionRequestParameters`](https://github.com/wevm/viem/blob/main/src/actions/wallet/prepareTransactionRequest.ts#L58-L66). [`sendTransaction`](https://github.com/wevm/wagmi/blob/main/packages/core/src/actions/sendTransaction.ts)  should be a good guide how to set this up (you'll also want to add a `test-d.ts` file so we can test for chain properties).

Reply:
thanks for the feedback, will add this shortly